packages/webcert/src/store/certificate/certificateActions.ts

```diff
@@ -169,6 +179,25 @@ const SET_VALIDATION_ERRORS_FOR_QUESTION = `${CERTIFICATE} Set validation errors
 const ADD_CLIENT_VALIDATION_ERROR = `${CERTIFICATE} Add client validation error`
 const REMOVE_CLIENT_VALIDATION_ERROR = `${CERTIFICATE} Remove client validation error`
 
+export interface CreateCertificateResponse {
+  certificateId: string
+}
+
+export interface CreateCertificate {
+  certificateType: string
+  patientId: string
+}
+
+export const createNewCertificate = createAction<CreateCertificate>(CREATE_NEW_CERTIFICATE)
+export const createNewCertificateStarted = createAction(CREATE_NEW_CERTIFICATE_STARTED)
+export const createNewCertificateSuccess = createAction<CreateCertificateResponse>(CREATE_NEW_CERTIFICATE_SUCCESS)
+export const createNewCertificateError = createAction<string>(CREATE_NEW_CERTIFICATE_ERROR)
+export const updateCreatedCertificateId = createAction<string>(UPDATE_CERTIFICATE_ID)
+
+export const updateCreateCertificate = createAction<CreateCertificate>(UPDATE_CREATE_CERTIFICATE)
+
+export const updateNavigateToCertificate = createAction<boolean>(UPDATE_NAVIGATE_TO_CERTIFICATE)
```

Comment:
This isn't used and can be removed, right?

Reply:
Correct. I remove them.